diff --git a/examples/sudo.conf.in b/examples/sudo.conf.in
index bdd676c..6c514b5 100644
--- a/examples/sudo.conf.in
+++ b/examples/sudo.conf.in
@@ -11,9 +11,9 @@
 # The plugin_options are optional.
 #
 # The sudoers plugin is used by default if no Plugin lines are present.
-#Plugin sudoers_policy @sudoers_plugin@
-#Plugin sudoers_io @sudoers_plugin@
-#Plugin sudoers_audit @sudoers_plugin@
+Plugin sudoers_policy @sudoers_plugin@
+Plugin sudoers_io @sudoers_plugin@
+Plugin sudoers_audit @sudoers_plugin@
 
 #
 # Sudo askpass:
@@ -85,7 +85,7 @@
 # To aid in debugging sudo problems, you may wish to enable core
 # dumps by setting "disable_coredump" to false.
 #
-#Set disable_coredump false
+Set disable_coredump false
 
 #
 # User groups:
